app/views/questionnaire.py

```diff
@@ -445,7 +445,7 @@ def _get_front_end_navigation(answer_store, current_location, metadata):
     completed_blocks = get_completed_blocks(current_user)
     navigation = Navigation(g.schema_json, answer_store, metadata, completed_blocks)
     block_json = SchemaHelper.get_block_for_location(g.schema_json, current_location)
-    if block_json is not None and (block_json['type'] == 'Questionnaire' or block_json['type'] == 'Interstitial'):
+    if block_json is not None and block_json['type'] in ('Questionnaire', 'Interstitial', 'Confirmation'):
```

Comment:
I don't think we need to check if the `block is not None` as would have been validated before to check the user is on a valid block/location